web_sdk/flutter_kernel_sdk.dart

```diff
@@ -2,16 +2,24 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+ // ignore_for_file: uri_does_not_exist
```

Comment:
Why don't the import uris exist?

Reply:
They do, provided you use the .packages file from third part, but locally the analyzer doesn't find it and that blows up on this file

Reply:
* third_party

Reply:
Could you add that in a comment here? Thanks!

Reply:
Done